src/Files.App/ViewModels/SelectedItemsPropertiesViewModel.cs

```diff
@@ -602,5 +602,23 @@ public bool IsHidden
 			get => isHidden;
 			set => SetProperty(ref isHidden, value);
 		}
+
+		private bool runAsAdmin;
+		public bool RunAsAdmin
+		{
+			get => runAsAdmin;
+			set
+			{
+				RunAsAdminEnabled = true;
+				SetProperty(ref runAsAdmin, value);
+			}
+		}
+
+		private bool runAsAdminEnabled;
+		public bool RunAsAdminEnabled
```

Comment:
What's this value for?

Reply:
Enabling the checkbox

Reply:
Should probably be used to hide the toggle switch?

Reply:
I think it was done to match the "read only" and "hidden" that get disabled and not hidden. But since the toggle has been moved to Shortut page perhaps it's better to hide it.

Reply:
Done